Sources/ScipioKit/ProjectGenerator.swift

```diff
@@ -49,52 +50,50 @@ struct ProjectGenerator {
         embedDebugSymbols isDebugSymbolsEmbedded: Bool,
         frameworkType: FrameworkType
     ) throws -> Result {
-        let projectPath = package.projectPath
+        let projectPath = AbsolutePath(package.projectPath.path)
 
         let project = try pbxproj(
```

Comment:
`pbxproj` still needs `AbsolutePath` and `TSCBasic.FileSystem`. However, this calling will be removed.